tests/functional/test_main.py

```diff
@@ -164,21 +164,31 @@ def test_song_search(app, test_db, persisted_user):
     assert b"Search for a song title" in response.data
 
 
-def test_song_search_results(client, app):
+def test_song_search_results(client, app, mocker):
     """Not sure if mock implemented properly?"""
+    mocker.patch(
+        "app.views.LastFMSearch.get_results",
+        return_value={
+            "result": {
+                "results": {
+                    "trackmatches": {
+                        "track": [
+                            {
+                                "name": "Army of Me",
+                                "artist": "björk",
+                                "url": "https://www.last.fm/music/Bj%C3%B6rk/_/Army+of+Me",
+                                "image": "/static/user_icon-01.png"
+                            }
+                        ]
+                    }
+                }
+            }
+        },
+    )
     response = client.get("/search-results?q=army of me")
-    LastFMSearch = Mock()
-
-    LastFMSearch.matches.return_value = [
-        {
-            "name": "Army of Me",
-            "artist": "Bj\xc3\xb6rk",
-            "url": "https://www.last.fm/music/Bj%C3%B6rk/_/Army+of+Me",
-        }
-    ]
-
+    
     assert response.status_code == HTTPStatus.OK
-    assert b"Bj\xc3\xb6rk" in response.data
+    assert b"bj\xc3\xb6rk" in response.data
```

Comment:
LOL I was confused what this was at first might be worth breaking this out into a local variable called bjork_name_bytes